platform/ios/docs/guides/Data-Driven Styling.md

```diff
@@ -0,0 +1,127 @@
+<!---
+To do - JK:
+- [ ] Fill it out with more detailed explanations
+- [ ] Add examples for camera or composite style functions
+- [ ] Cluster earthquakes as an example for camera function?
+- [ ] Clean up & optimize screenshots - replace with screenshots showing circles below symbol layer
+- [ ] Link API docs
+---->
+
+
+
+##Data-Driven Styling
+
+Mapbox's data-driven styling features allow you to use data properties to style your maps. You can style objects within the same layer differently based on their individual attributes. This enables you to style icons, routes, parks, and more based on attributes.
+
+![available bikes](img/data-driven-styling/citibikes.png)
+
+
+_Insert cool overview of things you can do!_
+
+### How to use Data-Driven Styling
+This guide uses earthquake data from the [U.S. Geological Survey](https://earthquake.usgs.gov/earthquakes/feed/v1.0/geojson.php) to illustrate to style a map based on attributes. For more information about how to work with GeoJSON data in our iOS SDK, please see our [working with GeoJSON data](https://www.mapbox.com/ios-sdk/api/3.5.0-beta.1/working-with-geojson-data.html) guide.
+
+`MGLStyleFunction`
+
+There are three subclasses of `MGLStyleFunction`:
+
+* `MGLCameraStyleFunction` - For a style value that changes with zoom level. For example, you can make the radius of a circle to increase based on zoom level. In iOS SDK v3.4, a `MGLStyleFunction` referred to what is now a `MGLCameraStyleFunction`.
```

Comment:
> referred to what is now a `MGLCameraStyleFunction`.

This seems like unnecessary repetition. Could we say something like: “In iOS SDK v3.4, this function was called `MGLStyleFunction`.”?